src/Transaction.proto

```diff
@@ -21,105 +21,66 @@ syntax = "proto3";
 
 option java_multiple_files = true;
 
+import "Signature.proto";
+import "Cryptography.proto";
 import "google/protobuf/timestamp.proto";
 
 package Catalyst.Protocol.Transaction;
 
 enum TransactionType {
-	NORMAL = 0;
-	CONFIDENTIAL = 1;
+	NORMAL = 0; // Public transaction type.
```

Comment:
the AccountType and SignatureType use the words `Confidential` and `Public`, I think here too we should go for that rather than `Normal`